apps/api/src/expenses/expenses.controller.ts

```diff
@@ -22,12 +34,16 @@ export class ExpensesController {
   }
 
   @Post('create-expense')
+  @UseInterceptors(FilesInterceptor('file', 5, { limits: { fileSize: 10485760 } })) //limit uploaded files to 5 at once and 10MB each
```

Comment:
Do both the `/create-expense` and `/upload-files` endpoints need the functionality of accepting files. Are they called independently?

Reply:
You are right. I cleaned it.